packages/element/src/platform.js

```diff
@@ -0,0 +1,9 @@
+const Platform = {
+	OS: 'web',
+	select: ( obj ) => ( 'web' in obj ? obj.web : obj.default ),
```

Comment:
What is this function for?

Reply:
It's to write this kind of code you can see here: https://facebook.github.io/react-native/docs/platform-specific-code#docsNav

Reply:
This is one of the techniques used in the cross-platform projects to avoid extracting very small utilities to their own file just to override them depending on the platform. 

@SergioEstevao - can you find a place in code where it could be used so we had a real-life example.

Reply:
@mkevins was sharing an example from gallery this morning, where the placeholder label would be different since we don't support drag and drop:

```jsx
const placeholderLabel = Platform.select({
    native: __( 'Add media' ),
    web: __( 'Drag images, upload new ones or select files from your library.' ),
} );
```

Reply:
I spoke too soon, I think `Platform.select` [doesn't support](https://github.com/facebook/react-native/blob/master/Libraries/Utilities/Platform.ios.js#L62) the `native` specifier. It would be a nice addition to our implementation.

Reply:
You want the native option to be added for the web component and the native component too, correct?

Reply:
@koke something like this for native: 

```
import { Platform as OriginalPlatform} from 'react-native';

const Platform = {
	OS: 'native',
	select: ( obj ) => ( 'native' in obj ? obj.native : OriginalPlatform.select(obj) ),
};

export default Platform;```

Reply:
Nice :smiley: :+1:

Reply:
I think `Platform.OS` should always return the more specific value (`ios` or `android`). For select to work, I think it might be easier to just add our own implementation for both platforms (and/or submit a PR to react-native). The logic for iOS would be:

```js
const Platform = {
    OS: 'ios',
    select: (spec): => 'ios' in spec ? spec.ios : 'native' in spec ? spec.native : spec.default,
}
```

`Platform` exposes other things like OS version, so we should still inherit those from the react-native implementation.

We could also not implement `native` and just use `web`/`default` to differentiate, but I think that would break expectations since we're very used to having `.native.js` files now.

Reply:
On my code above can I do this: 

`OriginalPlatform.select = (spec): => 'ios' in spec ? spec.ios : 'native' in spec ? spec.native : spec.default`